--- annotated.meta	2023-12-27 13:00:29.061991775 +0100
+++ test.meta	2023-12-27 12:59:14.375328088 +0100
@@ -1,22 +1,27 @@
-Contributor                     : Test Contributor
-Created                         : 2023-12-25
+Author                          : 
+Conformance                     : B
+Create Date                     : 2023:12:25 17:14:41+01:00
+Creator                         : 
+Creator Tool                    : ocrmypdf 16.0.2 / Tesseract OCR-PDF 5.3.3
 Directory                       : .
+Document ID                     : uuid:4224de48-db5d-11f9-0000-daf39fd7444b
 ExifTool Version Number         : 12.70
-File Access Date/Time           : 2023:12:25 17:08:45+01:00
-File Inode Change Date/Time     : 2023:12:25 17:08:39+01:00
-File Modification Date/Time     : 2023:12:25 17:08:39+01:00
-File Name                       : annotated.pdf
+File Access Date/Time           : 2023:12:27 12:58:05+01:00
+File Inode Change Date/Time     : 2023:12:25 17:14:41+01:00
+File Modification Date/Time     : 2023:12:25 17:14:41+01:00
+File Name                       : test.pdf
 File Permissions                : -rw-r--r--
-File Size                       : 14 kB
+File Size                       : 6.7 kB
 File Type Extension             : pdf
 File Type                       : PDF
+Format                          : application/pdf
+Language                        : en
 Linearized                      : No
-Metadata Date                   : 2023:12:25 16:08:39.337381+00:00
+Metadata Date                   : 2023:12:25 16:14:41.291096+00:00
 MIME Type                       : application/pdf
+Modify Date                     : 2023:12:25 16:14:41+00:00
 Page Count                      : 1
-PDF Version                     : 1.5
+Part                            : 2
+PDF Version                     : 1.7
 Producer                        : pikepdf 8.10.1
-PTEX Fullbanner                 : This is pdfTeX, Version 3.141592653-2.6-1.40.25 (TeX Live 2023/Arch Linux) kpathsea version 6.3.5
-Subject                         : Subject
-Trapped                         : False
-XMP Toolkit                     : pikepdf
+XMP Toolkit                     : XMP toolkit 2.9.1-13, framework 1.6
